### ISSUE TITLE
F-Droid can't build

### ISSUE THREAD

**Issue by licaon-kter** [TheZupZup/Linthra#303]
### What happened?

https://gitlab.com/fdroid/fdroiddata/-/jobs/15697342648#L1114 umm?

### Steps to reproduce

-

### Diagnostics

```text

```

### Install source

Other / not sure

### What were you using?

- [ ] Local files
- [ ] Jellyfin
- [ ] Navidrome / Subsonic
- [ ] Android Auto
- [ ] Chromecast / Cast
- [ ] Offline cache / downloads

### Logs (optional)

```text

```

**Comment by TheZupZup** [comment]
Hi @licaon-kter,

I’m sorry about this. The automatic update for v0.1.12 was triggered before I had verified the full release-only Android build path, and the tag failed during `lintVitalRelease` before producing any APK or AAB.

Please discard/ignore the automatic v0.1.12 update request on the F-Droid side. That tag is immutable and should not be built again.

I fixed the root cause in #304 by correcting the Android backup-rule XML and added `./gradlew lintVitalRelease` to pull-request CI so this type of release-only failure is caught before a stable tag is created.

The corrected release is v0.1.13 (`versionCode 113999`). It contains the same intended clean-reinstall protection plus the F-Droid build fix. Normal in-place updates still preserve user data.

Thank you for reporting it, and sorry again for the unnecessary failed autobuild and extra work.

**Comment by licaon-kter** [comment]
thanks
